### PR TITLE
Docs: Update README.md to be more concise 

### PR DESCRIPTION
**Pull request by Pcharlesme**
## Summary

Styling and formatting updates for the main README.

## Test Plan

N/A

### CODE REVIEW

**Comment by gmalouf** [comment]
Hi @Pcharlesme ,

Appreciate the work put in here. We will not be adding/accepting a logo at this time to be included in the project. if you want to strip that content out, we can probably get this PR to be mergeable.

**Comment by gmalouf** [comment]
Also important to flesh out the PR description of notable changes made.

**Review by algorandskiy — CHANGES_REQUESTED** [review]
Agreed with Gary - that external logo is not goer. Textural changes looks OK tho.

**Comment by Pcharlesme** [comment]
@algorandskiy @gmalouf Thanks for your feedback 
Modification has been made as requested 
Notable changes are 
Detailed Node Running Instructions , Test Section
Detailed Sections on Interfacing, Deployment, and Utilities:

File: README.md [hunk 1]
```diff
@@ -1,204 +1,173 @@
...
+
 [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](COPYING)
 
-Please see the [COPYING_FAQ](COPYING_FAQ) for details about how to apply our license.
+Please see the [COPYING_FAQ](COPYING_FAQ) for details on how to apply our license.
 
 Copyright (C) 2019-2024, Algorand Inc.
-
-[developer site url]: https://developer.algorand.org/
+```
```
[the remark this answers is not in the document]
Comment (Pcharlesme) [reply README.md]: File Updated

[the remark this answers is not in the document]
Comment (Pcharlesme) [reply README.md]: @cce

**Review by gmalouf — APPROVED** [review]

**Comment by gmalouf** [comment]
@Pcharlesme try rebasing to get latest from master/get build passing. README itself is looking good

**Pull request merged**